examples/tf-native/alibaba/redis/redis.tf

```diff
@@ -1,5 +1,5 @@
 module "redis" {
-  source               = "terraform-alicloud-modules/redis/alicloud"
+  source               = "github.com/chivalryq/terraform-alicloud-redis"
```

Comment:
Now please move the whole .tf to https://github.com/kubevela-contrib/terraform-modules/tree/master/alibaba and reference it in the Configuration.